frontend/public/components/app.jsx

```diff
@@ -133,23 +134,25 @@ class App extends React.PureComponent {
       <>
         <Helmet titleTemplate={`%s · ${productName}`} defaultTitle={productName} />
         <ConsoleNotifier location="BannerTop" />
-        <Page
-          header={<Masthead onNavToggle={this._onNavToggle} />}
-          sidebar={
-            <Navigation
-              isNavOpen={isNavOpen}
-              onNavSelect={this._onNavSelect}
-              onPerspectiveSelected={this._onNavSelect}
-            />
-          }
-        >
-          <ConnectedNotificationDrawer
-            isDesktop={isDrawerInline}
-            onDrawerChange={this._onNotificationDrawerToggle}
+        <ConnectedGuidedTourDrawer>
```

Comment:
I am not sure, this should be part of app.jsx or lazy loaded with individual components where required.
cc: @christianvogt @rohitkrai03